src/app/study-sequence/study-sequence.component.scss

```diff
@@ -0,0 +1,43 @@
+.back-to-view{
+    border-style: dashed;
+    height: 44px;
+}
+
+.container{
+    display: grid;
+    //Makes a grid with 3 columns
+    grid-template-columns: auto auto auto;
```

Comment:
Can you change the third column to be 30% instead of auto. It is currently too large.